install_files/ansible-base/group_vars/securedrop_application_server.yml

```diff
@@ -45,6 +45,9 @@ local_deb_packages:
 
 ### Used by the app role ###
 
+# Custom text on source interface
+custom_notification: "{{ custom_notification_text }}"
```

Comment:
Why the wrapper var here? Looks like this should simply be `custom_notification_text: ""`.